ui/raidboss/data/07-dt/raid/r1n.ts

```diff
@@ -1,14 +1,300 @@
+    ...mapEffectTileState,
+  },
+  '0D': {
+    'location': '0D',
+    ...mapEffectTileState,
+  },
+  '0E': {
+    'location': '0E',
+    ...mapEffectTileState,
+  },
+  '0F': {
+    'location': '0F',
+    ...mapEffectTileState,
+  },
+  '10': {
+    'location': '10',
+    ...mapEffectTileOverlay,
+  },
+  '11': {
+    'location': '11',
+    ...mapEffectTileOverlay,
+  },
+  '12': {
+    'location': '12',
+    ...mapEffectTileOverlay,
+  },
+  '13': {
+    'location': '13',
+    ...mapEffectTileOverlay,
+  },
+  '14': {
+    'location': '14',
+    ...mapEffectTileOverlay,
+  },
+  '15': {
+    'location': '15',
+    ...mapEffectTileOverlay,
+  },
+  '16': {
+    'location': '16',
+    ...mapEffectTileOverlay,
+  },
+  '17': {
+    'location': '17',
+    ...mapEffectTileOverlay,
+  },
+  '18': {
+    'location': '18',
+    ...mapEffectTileOverlay,
+  },
+  '19': {
+    'location': '19',
+    ...mapEffectTileOverlay,
+  },
+  '1A': {
+    'location': '1A',
+    ...mapEffectTileOverlay,
+  },
+  '1B': {
+    'location': '1B',
+    ...mapEffectTileOverlay,
+  },
+  '1C': {
+    'location': '1C',
+    ...mapEffectTileOverlay,
+  },
+  '1D': {
+    'location': '1D',
+    ...mapEffectTileOverlay,
+  },
+  '1E': {
+    'location': '1E',
+    ...mapEffectTileOverlay,
+  },
+  '1F': {
+    'location': '1F',
+    ...mapEffectTileOverlay,
+  },
+} as const;
+console.assert(mapEffectData);
+
+const headMarkerData = {
+  // Vfx Path: com_share1f
+  stack: '5D',
+  // Vfx Path: tank_lockon02k1
+  tankBuster: 'DA',
+  // Vfx Path: loc05sp_05a_se_p
+  lineStack: '178',
+} as const;
+console.assert(headMarkerData);
+
+// TODO:
+// Mouser
+// Predaceous Pounce
+// Leaping Black Cat Crossing
+
 const triggerSet: TriggerSet<Data> = {
   id: 'AacLightHeavyweightM1',
   zoneId: ZoneId.AacLightHeavyweightM1,
   timelineFile: 'r1n.txt',
-  triggers: [],
+  triggers: [
+    {
+      id: 'R1N One-two Paw Right Left',
+      type: 'StartsUsing',
+      netRegex: { id: '9309', source: 'Black Cat', capture: false },
+      durationSeconds: 9.5,
+      response: Responses.goRightThenLeft(),
+    },
+    {
+      id: 'R1N One-two Paw Left Right',
+      type: 'StartsUsing',
+      netRegex: { id: '930C', source: 'Black Cat', capture: false },
+      durationSeconds: 9.5,
+      response: Responses.goLeftThenRight(),
+    },
+    {
+      id: 'R1N Black Cat Crossing',
+      type: 'StartsUsing',
+      netRegex: { id: '930F', source: 'Black Cat', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Cards => Intercards',
+        },
+      },
+    },
+    {
+      id: 'R1N Elevate and Eviscerate',
+      type: 'StartsUsing',
+      netRegex: { id: '9317', source: 'Black Cat', capture: true },
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Aim for pristine tile',
```

Comment:
Is "pristine" directly used visibly in-game? If not, maybe we could go with "unbroken"?

Reply:
No, pristine doesn't appear in game for this mechanic.

I don't really like `unbroken` because there could be some confusion with regards to the cracked tiles (because they're just cracked, not broken). Maybe `uncracked` instead, or some other word that makes sense but doesn't have potential confusion?

Reply:
Mm, yes, `uncracked` could work. Mostly I'm thinking here of players who might not have "pristine" in the front of their minds and would be slow to process it.

Reply:
> Maybe `uncracked` instead, or some other word that makes sense but doesn't have potential confusion?

How about 'intact'?